workflow/scripts/score.R

```diff
@@ -55,7 +55,7 @@ if(as.numeric(df[1,2])>as.numeric(df[1,3]) & as.numeric(df[1,3])>as.numeric(df[1
 
 
 #PMD SCORES
-df<-try(read.delim(paste0(out_dir,"/",RefID,".PMDscores.txt"),header=FALSE,sep="\t"),silent=TRUE)
+df<-try(read.delim(paste0(output_dir,"/",RefID,".PMDscores.txt"),header=FALSE,sep="\t"),silent=TRUE)
```

Comment:
A general comment:  you could use the function `file.path` to generate path names. Then you wouldn't need to paste the separator "/"; in fact the separator would be inferred from the os. Also, from the docs: 

>      The implementation is designed to be fast (faster than paste) as
>     this function is used extensively in R itself.